src/index.js

```diff
@@ -1,19 +1,7 @@
-import humane from 'wavded/humane-js';
-import {inject} from 'aurelia-framework';
-import {I18N} from 'aurelia-i18n';
+import {Notification} from './notification';
+import {Config} from './config';
+export {Notification};
```

Comment:
You're not using `Notification` in this file, so you could combine this.

`export {Notification} from './notification';`